pkg/storage/stores/shipper/compactor/deletion/delete_requests_manager.go

```diff
@@ -195,24 +195,11 @@ func (d *DeleteRequestsManager) MarkPhaseFinished() {
 	}
 }
 
-func (d *DeleteRequestsManager) IntervalHasExpiredChunks(interval model.Interval) bool {
+func (d *DeleteRequestsManager) IntervalHasExpiredChunks(_ model.Interval) bool {
```

Comment:
Is this method still correctly named, rather than something like `IntervalMayHaveExpiredChunks`?

Reply:
Sounds good to me. Pushed the changes!